src/components/new-request/business-lookup.vue

```diff
@@ -65,6 +65,9 @@ enum States {
  */
 @Component({})
 export default class BusinessLookup extends Vue {
+  // Status of businesses to search for prop
+  @Prop({ default: 'ACTIVE' }) readonly businessStatus!: string
```

Comment:
I've set the default to 'Active' since this what is generally used and hence other Devs have no need to change anything.

Reply:
Can this of type `EntityStates`? (and cascade this type to code below)

Reply:
Also I suggest renaming this `searchStatus` as that's what it is (not a business status).

Reply:
I was debating on whether I should create an enum for them or not. Apparently, there's already one. The weird thing is I remember searching if there was one. Anyway, thanks for pointing this out Sev! I'll be making this change 🙌

Reply:
Use of enum + new prop name looks good. OK to resolve.